x-pack/plugins/ml/public/application/components/custom_urls/custom_urls.tsx

```diff
@@ -69,14 +87,16 @@ class CustomUrlsUI extends Component<CustomUrlsProps, CustomUrlsState> {
       dataViewListItems: [],
       queryEntityFieldNames: [],
       editorOpen: false,
+      supportedFilterFields: [],
     };
   }
 
   static getDerivedStateFromProps(props: CustomUrlsProps) {
     return {
       job: props.job,
       customUrls: props.jobCustomUrls,
-      queryEntityFieldNames: getQueryEntityFieldNames(props.job),
+      // For DFA uses the destination index Data View to get the query entities and falls back to source index Data View.
```

Comment:
Is this correct? When I deleted the dest data view, it no longer gives me the entity dropdown, so looks like it isn't falling back to the source index data view.

Reply:
Good catch! Fixed in 5b70da1b2fa75a6287841fa875d87c64387d1af6